core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -248,6 +248,7 @@ registry.ci.openshift.org/origin/4.11:prometheus-alertmanager quay.io/openshift/
 registry.ci.openshift.org/origin/4.11:prometheus-config-reloader quay.io/openshift/origin-prometheus-config-reloader:4.11 quay.io/openshift/origin-prometheus-config-reloader:4.11.0 quay.io/openshift/origin-prometheus-config-reloader:latest
 registry.ci.openshift.org/origin/4.11:prometheus-node-exporter quay.io/openshift/origin-prometheus-node-exporter:4.11 quay.io/openshift/origin-prometheus-node-exporter:4.11.0 quay.io/openshift/origin-prometheus-node-exporter:latest
 registry.ci.openshift.org/origin/4.11:prometheus-operator quay.io/openshift/origin-prometheus-operator:4.11 quay.io/openshift/origin-prometheus-operator:4.11.0 quay.io/openshift/origin-prometheus-operator:latest
+registry.ci.openshift.org/origin/4.11:prometheus-operator-admission-webhook quay.io/openshift/origin-prometheus-operator-admission-webhook:4.11 quay.io/openshift/origin-prometheus-operator-admission-webhook:4.11.0 quay.io/openshift/origin-prometheus-operator-admission-webhook:latest
```

Comment:
name looks ok, assuming we decide it really makes sense to create a separate image for this binary vs embedding it in an existing image.